main.ts

```diff
@@ -145,13 +145,13 @@ try {
               {
                 label: 'AzerothCore Repository',
                 click: () => {
-                  shell.openExternal('https://github.com/AzerothCore/AzerothCore/');
+                  shell.openExternal('https://github.com/AzerothCore/AzerothCore');
```

Comment:
it's `https://github.com/azerothcore/azerothcore-wotlk`

Reply:
ah right,